### DISCUSSION TITLE
Contract not showing in the Deploy section after having compiled - ZKSync Test Network

### DISCUSSION THREAD

**Discussion by vjgaur** [Cyfrin/foundry-full-course-cu#2102] [Q&A] [answered]
<img width="786" alt="Screenshot 2024-07-25 at 2 19 17 PM" src="https://github.com/user-attachments/assets/c4331047-4aad-48e8-ad69-901ff2a2a1be">

I have a problem , after compiling Simple Storage to Zksync network, I do not see anything under the deploy section. though the compile shows success with green tick mark.  But deploy section shows up empty. 
I have attached the screenshot for the reference.

can someone points me whats wrong ? what contracts are not showing in deploy section ?

**Comment by EngrPips** [comment]
Hello @vjgaur, do you have your contract file inside a contracts folder?

**Comment by vjgaur** [comment]
<img width="409" alt="Screenshot 2024-07-25 at 2 37 44 PM" src="https://github.com/user-attachments/assets/717364be-1735-486c-8c09-ef37d3df5c57">

Yes you can see the attached screenshot

**Reply by EngrPips** [reply]
Please switch your workspace to `default_workspace` and do everything in there, and let's see if you would still face the same issue 

<img width="1440" alt="Screenshot 2024-07-25 at 10 09 35" src="https://github.com/user-attachments/assets/863930a9-ba63-4ca2-b3c0-5e571179da92">

**Reply by EngrPips** [reply, marked as answer]
Please switch your workspace to `default_workspace` and do everything in there, and let's see if you would still face the same issue 

<img width="1440" alt="Screenshot 2024-07-25 at 10 09 35" src="https://github.com/user-attachments/assets/863930a9-ba63-4ca2-b3c0-5e571179da92">

**Comment by vjgaur** [comment]
Okay I did that, changed to default_workspace. It got the default project and now on compiling that on ZkSync network , its working as expected, it popped me that File Manager Permission window as well.  what I am supposed to do now ? Shall I move the SimpleStorage contract in default_workspace ?

**Reply by EngrPips** [reply]
Yeah, you can move the `SimpleStorage` to the `contracts` folder in the `default_workspace`

**Comment by iliyasali2107** [comment]
Hi everyone, i need a help. 
I changed to "default_workspace" and my contracts are in "contracts" folder, but in deploy tab there is still no ready contracts despite that compilation went successfully

And in terminal i have Warning saying that dependency AggregatorV3Interface.sol will not be used because it is not in contracts folder, but it is warning not an error and it didn't crush compilation, but anyway i checked by moving content of AggregatorV3Interface.sol to my FundMe.sol, and it didn't help to get my contract in deploy tab, i think Aggregator is not a problem

So i have no idea how to solve it, please help me

<img width="1397" alt="Снимок экрана 2024-07-26 в 06 55 58" src="https://github.com/user-attachments/assets/18053c8d-aeef-44f1-88ef-68a2e6497854">
<img width="1397" alt="Снимок экрана 2024-07-26 в 06 56 19" src="https://github.com/user-attachments/assets/cb2126b1-08e5-41d7-9bee-1c48ed7beba4">

**Reply by EngrPips** [reply]
Hello, start fresh and put every necessary file in the appropriate before compiling. Let's know if you still face the same issu.

**Reply by iliyasali2107** [reply]
Hello, i did everything like you wrote, but i encounter same problem, also tried to reactivate zkSync plugin, didn't help
I recorded a video how i was doing it: https://drive.google.com/file/d/1G8zZkrESYVPlVCjRad-yIuKotRdbZZin/view?usp=sharing

**Reply by EngrPips** [reply]
Why were you creating `default_workspace` folder yourself in the video that you shared? Isn't that supposed to be available by default

**Reply by iliyasali2107** [reply]
default_workspace that was available by default was used before, and in there it also didn't work

in this video i reseted everything, and using default ```default_workspace```: 
https://drive.google.com/file/d/1oqflFOYg31fQAXeBj0UG7IErX62QQo7h/view?usp=sharing

**Reply by EngrPips** [reply]
It should normally work like that. Maybe try another browser.
